tensorboard/uploader/proto/server_info.proto

```diff
@@ -74,8 +80,15 @@ message ExperimentUrlFormat {
   string id_placeholder = 2;
 }
 
+message PluginSpecification {
+  // Plugins for which the client wishes to upload data. These are plugin names
+  // as stored in the the `SummaryMetadata.plugin_data.plugin_name` proto
+  // field.
+  repeated string upload_plugins = 2;
+}
+
 message PluginControl {
-  // Only send data from plugins with these names. These are plugin names as
+  // Plugins for which data should be uploaded. These are plugin names as
```

Comment:
This is perhaps unrelated to this PR. But eventually we'll need a field to tell us which plugins are recently launched right? That way the uploader will be able to decide when to print a "new plugin" notification message. The alternative is to store the information in the uploader itself.